source/_integrations/google_assistant_sdk.markdown

```diff
@@ -11,6 +11,7 @@ ha_domain: google_assistant_sdk
 ha_codeowners:
   - '@tronikos'
 ha_integration_type: service
+ha_quality_scale: gold
```

Comment:
Do the [removal instructions](https://developers.home-assistant.io/docs/core/integration-quality-scale/rules) apply for this integration? 
There's a section in the [template](https://github.com/home-assistant/home-assistant.io/blob/current/source/_integrations/_integration_docs_template.markdown?plain=1#L200) you can use if it works for this integration. Please add additional steps, if needed. (or remove)

Reply:
It has removal instructions, see https://www.home-assistant.io/integrations/google_assistant_sdk#removing-the-integration

Reply:
Ooh, apologies. Not sure how I managed to overlook them 🙈